src/opm/output/eclipse/AggregateActionxData.cpp

```diff
@@ -370,6 +369,9 @@ const std::map<cmp_enum, int> cmpToIndex = {
             */
             std::size_t ind = 0;
             int noEPZacn = 26;
+            bool insideParen = false;
+            bool parenFirstCond = false;
+            bool allPrevLogicOp_AND = false;
             // write out the schedule Actionx conditions
             const auto& actx_cond = actx.conditions();
             for (auto cond_it = actx_cond.begin(); cond_it < actx_cond.end(); cond_it++) {
```

Comment:
Is there a reason why you don't use range based for:
```C++
for (const auto& cond : actx.conditions()) {
     ...
}
```

Reply:
Agree - done!